copilot_proxy/Dockerfile

```diff
@@ -8,6 +8,6 @@ RUN pip3 install --no-cache-dir -r requirements.txt
 
 COPY copilot_proxy .
 
-EXPOSE 5000
+# EXPOSE 5000
```

Comment:
This doesn't actually do anything, its just an instruction for users to know which port should be exposed.

Reply:
I think it's kind of pointless for a compose that can change ports at will, of course, it's worth discussing.

Reply:
Please see https://docs.docker.com/engine/reference/builder/#expose, it's noted as a documentation feature. I think in any case it's good to include it. Compose can change the ports from the post, not the ports from the application.

Reply:
I've read the documentation, but I've thought about it again. `app.py` does specify 5000 as the default, so the idea may be acceptable.🤔